internal/store/postgres/lineage_repository.go

```diff
@@ -283,18 +285,18 @@ func (repo *LineageRepository) buildDownstreamQuery(urn string, level int) (quer
 	return repo.buildRecursiveQuery(alias, nonRecursiveBuilder, recursiveBuilder)
 }
 
-func (repo *LineageRepository) buildRecursiveQuery(alias string, nonRecursiveBuilder, recursiveBuilder sq.SelectBuilder) (query string, args []interface{}, err error) {
-
+func (*LineageRepository) buildRecursiveQuery(alias string, nonRecursiveBuilder, recursiveBuilder sq.SelectBuilder) (
+	query string, args []interface{}, err error,
```

Comment:
named returns still present

Reply:
Named returns are meant to be used as documentation when there are multiple values with types that don't immediately make their meaning clear. So this is intentional.